templates/build/news-feed-filters.build.hbs

```diff
@@ -14,8 +14,27 @@
         <div id="collapse-\{{id}}-\{{@index}}" class="panel-collapse collapse">
           <div class="panel-body">
             <fieldset data-filter-group>
-              \{{#each [data]}}
-              <div class="btn hidden-filter-controls-filter focus-outline" tabindex="0" data-toggle="\{{[class]}}" data-field="\{{../name}}" data-value="\{{[name]}}" >\{{[name]}}</div>
+              \{{#each  ../detailViewOptions}}
+                \{{#ifCond type '==' 'date'}}
+                  \{{#each ../data}}
+                    \{{#ifCond ../../name '==' 'Date'}}
```

Comment:
Why are we checking for the "Date" name? I think the only requirement is for the type to be a date.

Reply:
We checking for type from an additional field which is in the detail view section, and then we check for the Date filter from the filters setup. It was described here:
![image](https://user-images.githubusercontent.com/85696424/128862983-2cb72875-ae19-4ba8-8c27-c0e796df28c3.png)

Reply:
Based on [this markup](https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/0179acd511a85a05c6f19c3ead98a64674397b67/templates/interface/detail-view-panels.interface.hbs#L27-L36) and [this code](https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/0179acd511a85a05c6f19c3ead98a64674397b67/js/layout-javascript/news-feed-code.js#L2381-L2387), it suggests that the detail view options are saved with a `type` property.

Changing the **Field type** dropdown should not lead to any change in any `name` property. If you find that is the case, please direct us to the code where a `name` property is being changed by updating the dropdown selection.